nf_core/pipeline-template/lib/NfcoreSchema.groovy

```diff
@@ -114,8 +121,7 @@ class NfcoreSchema {
             def params_ignore = params.schema_ignore_params.split(',') + 'schema_ignore_params'
             def expectedParamsLowerCase = expectedParams.collect{ it.replace("-", "").toLowerCase() }
             def specifiedParamLowerCase = specifiedParam.replace("-", "").toLowerCase()
-            def isCamelCaseBug = (specifiedParam.contains("-") && !expectedParams.contains(specifiedParam) && expectedParamsLowerCase.contains(specifiedParamLowerCase))
-            if (!expectedParams.contains(specifiedParam) && !params_ignore.contains(specifiedParam) && !isCamelCaseBug) {
+            if (!expectedParams.contains(specifiedParam) && !params_ignore.contains(specifiedParam) && !expectedParamsLowerCase.contains(specifiedParamLowerCase)) {
```

Comment:
That code was just recently introduces, I think that should stay, or is there a reason to delete?

Reply:
No idea. Could be an erroneous remnant from recent syncing with `dev` 🤔 Let me check.

Reply:
Hmmm...it is on [`dsl2_template`](https://github.com/nf-core/tools/blob/0e2aadc4f49b30de442478221138bf006ee7c1ae/nf_core/pipeline-template/lib/NfcoreSchema.groovy#L118) so I assume it's because I copied over the working version from #rnaseq 😏 #dsl2sycingbattles

Reply:
Fixed https://github.com/nf-core/tools/pull/1094/commits/eac411a6f8abdf98dfd24e298d203638d42ca12a